### DISCUSSION TITLE
adopt_solution takes a very long time

### DISCUSSION THREAD

**Discussion by danangsetyo5364** [phoebe-project/phoebe2#851] [Questions]
Hello! I was starting to learn PHOEBE for my thesis research. I would like to ask why sometimes it takes so long to run `b.adopt_solution()`. My light curve is semidetached type, and I am running `lc.geom` to estimates its initial parameters. Any idea about this problem? Thanks!

**Comment by aprsa** [comment]
@danangsetyo5364, can you please provide us with a minimal example to see if we can reproduce it?
